styles/styles.css

```
@@ -165,6 +211,23 @@ main {
     margin-right: auto
 }
 
+
+.advantages-photo-first {
+    flex-grow: 0;
+    flex: none;
```

```suggestion
.advantages-photo-first {
```

Ты сначала отменил flex-grow, а потом отменил ещё раз. 
Потом ты просто задашь блоку с текстом ширину и они с картинкой четко встанут в одну линию.